### PR TITLE
Updated contract bindings

### PR DESCRIPTION
**Pull request by yondonfu**
Updated contract bindings to match the latest develop branch in the protocol repo.

Removed the contract bytecode from the bindings, so now they just contain the contract ABI.

### CODE REVIEW

File: core/claimmanager.go [hunk 1]
```diff
@@ -220,7 +221,7 @@ func (c *BasicClaimManager) Claim() (claimCount int, rc chan types.Receipt, ec c
 		}
 
 		//Do the claim
-		go func(segRange [2]int64, rc chan types.Receipt, ec chan error) {
+		go func(rangeIdx int, segRange [2]int64, rc chan types.Receipt, ec chan error) {
```
Comment (yondonfu) [inline core/claimmanager.go:224]: Making sure that the goroutine closes over the value at each iteration of the loop

File: eth/client.go [hunk 2]
```diff
@@ -158,42 +150,20 @@ func NewClient(account accounts.Account, passphrase string, datadir string, back
 
 	transactOpts.GasPrice = gasPrice
 
-	token, err := contracts.NewLivepeerToken(tokenAddr, backend)
+	controller, err := contracts.NewController(controllerAddr, backend)
```
Comment (yondonfu) [inline eth/client.go:153]: Now we can just pass in the controller address and we can get every other contract address via the controller's registry

Comment (ericxtang) [reply eth/client.go:153]: 👍

File: core/claimmanager.go [hunk 3]
```diff
@@ -270,15 +271,16 @@ func (c *BasicClaimManager) Verify() error {
 		if shouldVerifySegment(segNo, scm.claimStart, scm.claimEnd, scm.claimBlkNum.Int64(), scm.claimBlkHash, verifyRate) {
 			glog.Infof("Calling verify")
 			//TODO: Add data to IPFS
```
Comment (ericxtang) [inline core/claimmanager.go]: Let's remove the "TODO"

**Review by dob — COMMENTED** [review]
Looks good to me. This has nothing to do with the updated bindings, but in general the choice parsing/matching in the CLI seems a little weird. We're treating all the integers as strings. Separate issue I guess.

File: core/claimmanager.go (hunk 1 above)
Comment (ericxtang) [reply core/claimmanager.go:224]: Is rangeIdx used anywhere?

Comment (yondonfu) [reply core/claimmanager.go:224]: Its used as the claimId https://github.com/livepeer/go-livepeer/blob/8c120b3ae219b23e0b6a644baab788fac75658b7/core/claimmanager.go#L244

**Review by ericxtang — APPROVED** [review]
🚢

**Pull request merged**